fluent-package/templates/package-scripts/fluent-package/deb/postinst

```diff
@@ -7,25 +7,31 @@ set -e
 prevver="$2"
 
 add_system_user() {
-    if ! getent passwd <%= compat_service_name %> >/dev/null; then
-        adduser --group --system --home /var/lib/<%= package_dir %> <%= compat_service_name %>
+    if ! getent passwd _<%= service_name %> >/dev/null; then
+	if ! getent group <%= compat_service_name %> >/dev/null; then
```

Comment:
Is it intentional that checking `group` instead of `passwd` here?

Reply:
It should be `passwd`.  thanks.